regression-exponential.d.ts

```diff
@@ -6,6 +6,9 @@ export interface ExponentialRegressionModel {
   B: number;
 }
 export declare class ExponentialRegression extends BaseRegression {
+  A: number;
+  B: number;
```

Comment:
A JSDoc comment with an explanation of what those numbers are would be helpful.

Reply:
Thanks for the suggestion. Added in [f3ef3cf](https://github.com/mljs/regression-exponential/pull/24/commits/f3ef3cf458a0eaca7f3a9a0a2ab4345b21cbd915)

Reply:
Wait, I just found that the README is wrong. A is in fact B.